### DISCUSSION TITLE
Capacitor can works with ionic 6 and react js

### DISCUSSION THREAD

**Discussion by jschmucke** [timbru31/cordova-plugin-lottie-splashscreen#489] [Q&A]
Is this Capacitor compatible with reactjs? Is there a guide or example of that implementation for Android and iOS?

**Comment by timbru31** [comment]
Generally this shouldn't be a problem. Apache Cordova and Capactior work fine with React: https://capacitorjs.com/solution/react
Due to high maintenance requirement there is currently no example with Lottie.

**Reply by jschmucke** [reply]
Thank you for answering, 
But I can't get any example or information on how to use this plugin with ionic 6 reactjs, could you provide me with one.
thank you very much
